src/sbsign/sbsign.c

```
@@ -254,6 +254,10 @@ static int verb_sign(int argc, char *argv[], void *userdata) {
         if (dstfd < 0)
                 return log_error_errno(r, "Failed to open temporary file: %m");
 
+        r = fchmod_umask(dstfd, 0666);
+        if (r < 0)
+                log_debug_errno(r, "Failed to change temporary file mode: %m");
```

late, but I'd like to add `, ignoring` as the failure is ignored.